app.py

```diff
@@ -533,4 +542,4 @@ def reset_dropdown_enhance(_):
 
 # Running the server
 if __name__ == '__main__':
-    app.run_server(debug=True)
+    app.run_server(debug=True, port=8051)
```

Comment:
why is this required? Is this a DDS thing?

Reply:
A port setting for personal debugging. Will remove it.